examples/spark-sensors-proto/README.md

```diff
@@ -0,0 +1,253 @@
+|   src|gauge|             value|
++------+-----+------------------+
+|src-97|  oil| 11.28699131177144|
+|src-33|  gas| 54.13269242596257|
+|src-33|  gas| 988.3230454723247|
+|src-97|  gas| 148.3055826758501|
+|src-97|  gas| 148.3055826758501|
+|src-33|  gas| 54.13269242596257|
+|src-97|  oil|  760.024470736732|
+|src-97|  oil|  760.024470736732|
+|src-33|  gas| 988.3230454723247|
+|src-97|  gas|1042.3404196321565|
+|src-97|  gas|1042.3404196321565|
+|src-33|  oil| 27.75132596682821|
+|src-33|  gas| 976.6856203943967|
+|src-97|  gas| 385.3532592940638|
+|src-33|  gas| 976.6856203943967|
+|src-33|  oil| 27.75132596682821|
+|src-97|  gas| 385.3532592940638|
+|src-33|  gas|1269.5736118755415|
+|src-97|  oil|188.75469481408513|
+|src-33|  gas|1269.5736118755415|
++------+-----+------------------+
+only showing top 20 rows
+
+-------------------------------------------
+Batch: 2
+-------------------------------------------
++------+-----+------------------+
+|   src|gauge|             value|
++------+-----+------------------+
+|src-33|  oil| 847.6950249637855|
+|src-33|  gas| 894.1615286881954|
+|src-33|  oil| 847.6950249637855|
+|src-97|  gas| 901.3472093816678|
+|src-97|  gas| 1143.234750548319|
+|src-97|  gas| 901.3472093816678|
+|src-97|  gas| 1143.234750548319|
+|src-72|  oil|1571.1498223984206|
+|src-21|  oil|148.37603003222227|
+|src-72|  oil|1571.1498223984206|
+|src-21|  gas|453.74071025583896|
+|src-21|  gas|453.74071025583896|
+|src-21|  oil|148.37603003222227|
+|src-10|  gas|1649.6090844251842|
+|src-10|  gas| 178.6005146200035|
+|src-92|  oil| 376.5333753643078|
+|src-92|  oil| 376.5333753643078|
+|src-10|  gas|1649.6090844251842|
+|src-10|  gas| 178.6005146200035|
+|src-19|  gas|1547.4380898089883|
++------+-----+------------------+
+only showing top 20 rows
+```
+
+* Undeploy.
+
+```bash
+$ kubectl cloudflow undeploy spark-sensors
+```
+### Protobuf vs Avro
+
+This project demonstrates the usage of Protobuf for message encoding. Protobuf is enabled through usage of
+[Scala-PB Spark](https://scalapb.github.io/docs/sparksql/).
+ScalaPB comes with its own `implicits` - `scalapb.spark.Implicits._` which is not compatible with `cloudflow.spark.sql.SQLImplicits._`
+The following are the rules for using imports.
+
+* When protobufs are not used, you should use `cloudflow.spark.sql.SQLImplicits._` in all classes as required - see example
+* When protobufs are used, you should use `scalapb.spark.Implicits._`. Here are the caveats. 
+You may want to import `cloudflow.spark.sql.SQLImplicits.StringToColumn` to convert $"col name" into a Column.
+If you use custom scala classes in you Spark calculations, you need to explicitely create encoder for it. See
+[SparkRandomGenDataIngress](src/main/scala/sensors/proto/SparkRandomGenDataIngress.scala) for example
+* Usage of both imports in the same class will lead to a problem
+
+Also note dependency overwrites necessary to use the right version of `com.fasterxml.jackson.core` versions
```

Comment:
Why is this necessary?

Reply:
Was there before. Removed, based on changes to cloudflow here